### PR TITLE
Issue2244

### PR DESCRIPTION
**Pull request by diabagatekelly**
Fixes # 2244

### Description
Added Useful Links section at the bottom of Docker Page tutorial, as well as links to Docker Concepts, Docker Overview, Docker Compose, Why Use Docker?, Wikipedia page, and another tutorial.

### RawGit preview link

https://raw.githack.com/diabagatekelly/diabagatekelly.github.io/issue2244/#!pages/vi/vi-docker-tutorial.md

### CODE REVIEW

File: pages/vi/vi-docker-tutorial.md [hunk 1]
```diff
@@ -254,4 +254,13 @@ Commands:
   version            Show the Docker-Compose version information
 ```
 
+## Useful Links
+
+[Why install Docker?](https://www.docker.com/why-docker) 
+[Docker Concepts](https://docs.docker.com/get-started/#docker-concepts)
+[Docker Overview](https://docs.docker.com/engine/docker-overview/)
+[Docker Compose](https://docs.docker.com/compose/overview/)
+[Wikipedia page on Docker](https://en.wikipedia.org/wiki/Docker_(software)
```
Comment (lmmrssa) [inline pages/vi/vi-docker-tutorial.md]: this link does not have much information so we can remove this link

File: pages/vi/vi-docker-tutorial.md [hunk 2]
```diff
@@ -254,4 +254,13 @@ Commands:
   version            Show the Docker-Compose version information
 ```
 
+## Useful Links
+
+[Why install Docker?](https://www.docker.com/why-docker)
```
Comment (lmmrssa) [inline pages/vi/vi-docker-tutorial.md]: `https://www.docker.com/what-docker` gives idea of what docker is

File: pages/vi/vi-docker-tutorial.md [hunk 3]
```diff
@@ -254,4 +254,13 @@ Commands:
...
 ```
 
+## Useful Links
+
+[Why install Docker?](https://www.docker.com/why-docker) 
+[Docker Concepts](https://docs.docker.com/get-started/#docker-concepts)
+[Docker Overview](https://docs.docker.com/engine/docker-overview/)
+[Docker Compose](https://docs.docker.com/compose/overview/)
+[Wikipedia page on Docker](https://en.wikipedia.org/wiki/Docker_(software)
+[Other useful Docker tutorial](https://scotch.io/tutorials/getting-started-with-docker)
```
Comment (lmmrssa) [inline pages/vi/vi-docker-tutorial.md]: We already have added links with docker cli command so use that
`https://docs.docker.com/engine/reference/commandline/cli/`

**Comment by diabagatekelly** [comment]
@lmmrssa so if I understand correctly, I will change it to this?

[What is Docker?](https://www.docker.com/what-docker) 
[Docker Concepts](https://docs.docker.com/get-started/#docker-concepts)
[Docker Overview](https://docs.docker.com/engine/docker-overview/)
[Docker Compose](https://docs.docker.com/compose/overview/)
[Docker CLI Command](https://docs.docker.com/engine/reference/commandline/cli/)
[Docker Installation](http://open-learning-exchange.github.io/#!./pages/vi/vi-docker-installation.md)

**Comment by lmmrssa** [comment]
yes @diabagatekelly

File: pages/vi/vi-docker-tutorial.md (hunk 2 above)
Comment (diabagatekelly) [reply pages/vi/vi-docker-tutorial.md]: will do

File: pages/vi/vi-docker-tutorial.md (hunk 3 above)
Comment (diabagatekelly) [reply pages/vi/vi-docker-tutorial.md]: will do

**Comment by diabagatekelly** [comment]
I made the required changes.

**Pull request merged**